compiler/luci-interpreter/src/loader/KernelBuilder.cpp

```diff
@@ -626,6 +627,22 @@ std::unique_ptr<Kernel> KernelBuilder::visit(const luci::CircleOutput *)
   throw std::runtime_error("Output node cannot be executed.");
 }
 
+std::unique_ptr<Kernel> KernelBuilder::visit(const luci::CirclePack *node)
+{
```

Comment:
Can you add the assert?

`assert(node->arity() == node->values_count());`

Reply:
I will add it.